signalr-node-server/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
```

Comment:
I think this should be removed. Change logs are great for release notes, but this is what commits are for.